doc/projectBranding.md

```diff
@@ -102,3 +107,137 @@ Finally, if you'd rather not display any "About" pop-up and remove the associate
 // import { AboutContent } from '../../components/AboutContent';
 export const About = null;
 ```
+
+# Translations customization
+
+In order to edit translation labels, you can directly edit translation files :
+[public/locales/en/translation.json](../public/locales/en/translation.json) or/and
+[public/locales/fr/translation.json](../public/locales/fr/translation.json).
```

Comment:
Are we sure we want solutions to modify generic translation files ? The whole idea of _custom.json_ is to avoid it...

Reply:
Rephrased in a new commit :slightly_smiling_face: